app/api/domains/route.ts

```diff
@@ -104,3 +103,5 @@ export async function POST(request: Request) {
 
   return NextResponse.json(availableDomains);
 }
+
+export const runtime = "experimental-edge";
```

Comment:
Not sure if I need to export `config` here instead, or is `config` the old syntax?
```suggestion
export const config = {
  runtime: 'edge',
};
```

Reply:
`config` is for `pages/*` – you have the correct syntax.